### ISSUE TITLE
Investigate lack of email alert for 2023:203 bad point

### ISSUE THREAD

**Issue by taldcroft** [sot/fss_check#20]
There should have been an alert email to aca_alert associated with this bad point (which is likely a DBE, not real):
<img width="663" alt="image" src="https://github.com/sot/fss_check/assets/348089/41a84db2-889f-4daa-8c6c-785af84f7f93">

**Comment by jeanconn** [comment]
Though I note that the dev version that is running is still configured to send to taldcroft -- if it had sent something.

**Comment by taldcroft** [comment]
> Though I note that the dev version that is running is still configured to send to taldcroft -- if it had sent something.

@jeanconn - I'm not sure where you got that from.

But in the meantime I found the culprit. The `task_schedule.cfg` in #16 specifies the email as `aca_alert@cfa.harvard.edu` but it should be `aca_alert` because `@head.cfa.harvard.edu` is hardwired into `send_mail()`. The log showed this, where some munging happened:
```
<<2023-Jul-22 07:46>> 2023-07-22 06:46:35,569 send_mail: Sent mail to ['aca_alert.harvard.edu']
```
I guess the send mail doesn't really know if the mail got delivered.

**Comment by jeanconn** [comment]
Regarding "@jeanconn - I'm not sure where you got that from." 
Specifically, I got it from either of the two task_schedule.cfgs that are in /proj/sot/ska/data, but if I was supposed to get it from `/export/tom/miniconda3/envs/ska3-fss-check/lib/python3.10/site-packages/fss_check/task_schedule.cfg`, you're right, I missed that one.

**Comment by taldcroft** [comment]
I pushed some commits to #16 that should help.

**Comment by taldcroft** [comment]
Closed by #16
